src/main/scala/net/snowflake/spark/snowflake/SnowflakeRelation.scala

```diff
@@ -206,4 +208,8 @@ private[snowflake] case class SnowflakeRelation(
     val fieldMap = Map(schema.fields.map(x => x.name -> x): _*)
     new StructType(columns.map(name => fieldMap(name)))
   }
+
+  private def isQuoted(name: String): Boolean = {
+    name.startsWith("\"") && name.endsWith("\"")
```

Comment:
This should never happen since we always insert quotes around special names for DataFrame columns. If the user makes a mistake, then Spark would error-out during name resolution before the code reaches this point.